funding/manager.go

```diff
@@ -1398,11 +1398,16 @@ func (f *Manager) handleFundingOpen(peer lnpeer.Peer,
 	// the amount of the channel, and also if any funds are being pushed to
 	// us. If a depth value was set by our channel acceptor, we will use
 	// that value instead.
-	numConfsReq := f.cfg.NumRequiredConfs(msg.FundingAmount, msg.PushAmount)
+	remoteFeatures := peer.RemoteFeatures()
+	skipConfirmationOptional := remoteFeatures.HasFeature(
```

Comment:
Style nit: doesn't use typical line folding style.

Reply:
Can you please elaborate on what's wrong here? I had to cut the line so it won't exceed 80 characters.